docs/DafnyRef/README.md

```diff
@@ -101,4 +101,4 @@ file as well.
 ## LSP
 
 Many IDEs interact with Language Servers (via LSP). Possibly an LSP protocol
-will be generated in the future.
+will be generated in the future.
```

Comment:
I think this line is obsolete. We do have an LSP now...
No need to change this PR but next one dealing with this README should update this.